doc/kubevela-definitions/cardano/cardano-wallet-connector.md

```diff
@@ -0,0 +1,39 @@
+# Cardano Wallet Connector trait
+
+## Overview
+
+The Cardano Wallet Connector trait in KubeVela serves as a pass-through between your application and the Cardano wallet, using specific application characteristics and properties.
```

Comment:
What does "serves as a pass-through" mean here? What "specific application characteristics and properties"?

Reply:
this was too generic.

gonna write down in a simple way.